build/template-postbuild-code-analysis.yaml

```diff
@@ -6,11 +6,25 @@ steps:
   displayName: 'Run Roslyn Analyzers'
   inputs:
     userProvideBuildInfo: auto
+    suppressionFileForCompilerWarnings: 'microsoft-authentication-library-for-dotnet\RoslynSuppressions.txt'
+    msBuildCommandline: 'dotnet build'
+    rulesetName: Recommended
+    rulesetVersion: Latest
   env:
     system_accesstoken: $(System.AccessToken)
   continueOnError: true
+  condition: and(succeeded(), eq(variables['PipelineType'], 'OneBranch'))
```

Comment:
Wouldn't this always be false since this postbuild template is removed from OneBranch-libsandsamples yaml?

Reply:
Yes, not needed